metagpt/provider/anthropic_api.py

```diff
@@ -18,17 +18,29 @@ def ask(self, prompt):
         res = client.completions.create(
             model="claude-2",
             prompt=f"{anthropic.HUMAN_PROMPT} {prompt} {anthropic.AI_PROMPT}",
-            max_tokens_to_sample=1000,
+            max_tokens_to_sample=8192,
         )
         return res.completion
     
     async def aask(self, prompt):
-            client = Anthropic(api_key=CONFIG.claude_api_key)
-
-            res = client.completions.create(
-                model="claude-2",
-                prompt=f"{anthropic.HUMAN_PROMPT} {prompt} {anthropic.AI_PROMPT}",
-                max_tokens_to_sample=1000,
-            )
-            return res.completion
+        client = Anthropic(api_key=CONFIG.claude_api_key)
+
+        res = client.completions.create(
+            model="claude-2",
+            prompt=f"{anthropic.HUMAN_PROMPT} {prompt} {anthropic.AI_PROMPT}",
+            max_tokens_to_sample=1000,
+        )
+        return res.completion
+    
+    def sync_tokens(self, text):
+        client = Anthropic()
+        tokens = client.count_tokens(text)
+        return tokens
+
+
+
+    async def async_tokens(self, text):
```

Comment:
Provide the type description and comments for the input and output parameters. E.g. async def aask(self, prompt: str)